include/tvm/runtime/container/adt.h

```diff
@@ -136,7 +136,7 @@ class ADT : public ObjectRef {
    */
   template <typename... Args>
   static ADT Tuple(Args&&... args) {
-    return ADT(0, std::forward<Args>(args)...);
+    return ADT(0, {std::forward<Args>(args)...});
```

Comment:
would be great to first keep ADT inact. We should also consider moving to Array later

Reply:
it isn't changing ADT. Without brackets in some cases it doesn't correctly call into the constructor that accepts initializer list https://github.com/mlc-ai/relax/blob/a8f562ed2bfdf12486863e4b6f553e0e4b9d8f12/include/tvm/runtime/container/adt.h#L110
I can also send a PR upstream separately for this